src/_includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html

```diff
@@ -141,27 +137,38 @@
                    required>
         </div>
         <div class="form-l_col form-l_col-1">
-            <label class="form-label-header" for="complaint-desc">
-                Description of complaint
+           <label for="request-desc">
+                <span class="form-label-header" for="request-desc">
```

Comment:
Can this span include `display: block` style in some way so as to make the `<br>` that follows it unnecessary?